gstd/gstd_pipeline.c

```diff
@@ -301,6 +329,26 @@ gstd_pipeline_get_property (GObject * object,
           self->event_handler);
       g_value_set_object (value, self->event_handler);
       break;
+    case PROP_POSITION:
+      if (!gst_element_query_position (self->pipeline, GST_FORMAT_TIME, &self->position)) {
+        /* if the query could not be performed. return 0 */
+        self->position = 0;
+      }
+
+      GST_DEBUG_OBJECT (self, "Returning pipeline position %" GST_TIME_FORMAT,
+          GST_TIME_ARGS (self->position));
+      g_value_set_int64 (value, self->position);
+      break;
+    case PROP_DURATION:
+      if (!gst_element_query_duration (self->pipeline, GST_FORMAT_TIME, &self->duration)) {
+        /* if the query could not be performed. return 0 */
+        self->duration = 0;
```

Comment:
This is not portable, this will fail in imx6, por example. Use G_GINT64_CONSTANT(0) instead.